templates/index.html

```diff
@@ -69,10 +69,10 @@ <h3 class="fs-title">
     students.</p>
     
     <ul class="simple">
-    <li><a class="reference external" href="cass.oregonstate.edu/units/osuosl/">Open Source Lab</a>: Open source hosting and software development</li>
-    <li><a class="reference external" href="cass.oregonstate.edu/units/software-dev-group/">Software Development Group</a>: Designs and develops applications for all major
+    <li><a href="http://cass.oregonstate.edu/units/osuosl/">Open Source Lab</a>: Open source hosting and software development</li>
+    <li><a href="http://cass.oregonstate.edu/units/software-dev-group/">Software Development Group</a>: Designs and develops applications for all major
     platforms</li>
-    <li><a class="reference external" href="cass.oregonstate.edu/units/test-and-iot-lab/">Test and IoT Lab</a>: Hardware and software testing services and Internet of
+    <li><a href="http://cass.oregonstate.edu/units/test-and-iot-lab/">Test and IoT Lab</a>: Hardware and software testing services and Internet of
```

Comment:
@kelnera This doesn't actually link to anything. What do you suggest it gets changed to?